litmus-portal/frontend/src/views/ChaosWorkflows/BrowseTemplate/index.tsx

```diff
@@ -65,20 +88,7 @@ const BrowseAWorkflow: React.FC<BrowseTemplateProps> = ({ location }) => {
             <ButtonOutlined onClick={() => history.push('/workflows')}>
               <>Back</>
             </ButtonOutlined>
-            <ButtonFilled
-              variant="success"
-              onClick={() => {
-                workflowAction.setWorkflowDetails({
-                  description: '',
-                  isCustomWorkflow: false,
-                  customWorkflows: [],
-                });
-                history.push({
-                  pathname: '/create-workflow',
-                  search: `?projectID=${projectID}&projectRole=${userRole}`,
-                });
-              }}
-            >
+            <ButtonFilled variant="success" onClick={() => preSelectWorkflow()}>
               <>Schedule this template</>
             </ButtonFilled>
```

Comment:
Please add translations @S-ayanide

Reply:
Also remove the <>....</> this may not be required to provide text to the buttons

Reply:
Updated